pkg/cmd/pulumi/policy/policy_publish_test.go

```diff
@@ -116,3 +119,97 @@ func TestPolicyPublishCmd_orgNamePassedIn(t *testing.T) {
 	err := cmd.Run(context.Background(), lm, []string{"org1"})
 	require.NoError(t, err)
 }
+
+// TestPolicyPublishCmd_Metadata tests that vcs metadata is included with the publish command.
+func TestPolicyPublishCmd_Metadata(t *testing.T) {
+	t.Parallel()
+
+	tmpDir := t.TempDir()
+	subDir := filepath.Join(tmpDir, "subdirectory")
+	err := os.MkdirAll(subDir, 0o755)
+	require.NoError(t, err)
```

Comment:
Curious, couldn't this use `ptesting.NewEnvironment()`?  That way we don't have to invent a new `runCommand` function below